Thread: add step to make it preferred network (#42596)
home-assistant/home-assistant.github.io: source/_integrations/thread.markdown

   - Importing the credentials allows a Google- or Apple-created Thread network to be the preferred network of Home Assistant.
   <img width="400" src='/images/integrations/thread/thread-preferred-network.png'>

> 3. If a **Make preferred network** button becomes available, select it.

Reviewer: nit/optional: Just thought this might sound a bit more natural:
```suggestion
3. If the **Make preferred network** button is available, select it.
```

or even this:
```suggestion
3. If the **Make preferred network** button is shown, select it.
```

Author: love the options :smile: thx.

   - Importing the credentials allows a Google- or Apple-created Thread network to be the preferred network of Home Assistant.
   <img width="400" src='/images/integrations/thread/thread-preferred-network.png'>
3. If a **Make preferred network** button becomes available, select it.

>    - Joined networks can improve coverage and latency.

Reviewer: Hmm. I don’t really think this point is accurate. Like, we don’t really join a Thread network by making it preferred, so the coverage/latency aspect also isn’t that accurate here.

I’d probably just remove that line.

The main advantage of having a preferred network is that if you set up a new OTBR instances later, it’s automatically set up to be "part of the same Thread network".

Depending on if you also sent those Thread network credentials to your phone, the app will join new Thread devices to the network as well.

```suggestion
```

But HA essentially has access to devices on all Thread networks it sees, even if they're not preferred.
